src/utils/DarkModeSwitch.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react'
+import {useSpring, animated} from '@react-spring/web'
+
+export const defaultProperties = {
```

Comment:
Are you using this anywhere else? If not, I don't think you need to export it.

Reply:
ah Okay. removed the `export`.